oauth2/src/test/groovy/com/okta/spring/boot/oauth/env/OktaOAuth2PropertiesMappingEnvironmentPostProcessorTest.groovy

```diff
@@ -69,6 +75,31 @@ class OktaOAuth2PropertiesMappingEnvironmentPostProcessorTest {
         assertThat environment.getProperty(TOKEN_URI), nullValue()
         assertThat environment.getProperty(USER_INFO_URI), nullValue()
         assertThat environment.getProperty(PROVIDER_KEYS_URI), nullValue()
+        assertThat environment.getProperty(RS_CLIENT_ID), nullValue()
+        assertThat environment.getProperty(RS_CLIENT_SECRET), nullValue()
+        assertThat environment.getProperty(RS_INTROSPECTION_URI), nullValue()
+    }
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
```

Comment:
nit: don't use this pattern of expecting exceptions, unless the test has a single line.
In this case line 91 could throw the `IllegalArgumentException` and the test would still pass

Either use the old try/catch/fail, or the `expect IllegalArgumentException, { yourCodeHere() }` model, similar to what is used in the SDK and other projects: https://github.com/okta/okta-sdk-java/blob/master/impl/src/test/groovy/com/okta/sdk/impl/Util.groovy#L22

Reply:
noted.